Makefile

```diff
@@ -69,6 +69,9 @@ endif
 
 DOCKER_RUN_DOCKER := $(DOCKER_FLAGS) "$(DOCKER_IMAGE)"
 
+# shell command used for `make shell`
+SH := $(if $(SH),$(SH),bash)
```

Comment:
Just doing `SH := bash` here should be sufficient to allow it to be overwritten via `make shell SH='...'` :+1: